tests/__init__.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
```

Comment:
Adding this may have been unnecessary - it looked like a previous issue where files weren't being collected due to absence of __init__.py. I will try removing it.

Reply:
Seems necessary. [Failed](https://travis-ci.org/Azure/iotedgedev/builds/398020875) on every virtual env when I tried removing it again. Python isn't my usual world, so very possible I'm missing something else. If so, let me know.